batchflow/models/torch/base.py

```diff
@@ -341,6 +343,8 @@ def build(self):
         # Store some of the config values
         self.microbatch = self.full_config.get('microbatch', None)
         self.sync_frequency = self.full_config.get('sync_frequency', 1)
+        self.sam_parameter_placeholder = self.full_config.get('sam_parameter_placeholder', False)
```

Comment:
Why not just `sam` for brevity?

Reply:
what about just having sam_rho == 0 for no SAM?